.gitignore

```diff
@@ -0,0 +1,12 @@
+*~
```

Comment:
Checkout https://github.com/github/gitignore/blob/main/Python.gitignore as comprehensive `.gitignore` for Python projects.

Reply:
Do we just want to use that without modifications?